content/misc/misc-12/contents.lr

```diff
@@ -5,7 +5,8 @@ title: How can I share files anonymously through Tor?
 seo_slug: share-files-anonymously-through-tor
 ---
 description:
+For sharing files over Tor, <mark><a href="https://onionshare.org/">OnionShare</a></mark> is a good option. OnionShare is temporarily/stealthily made available as a Tor Onion Service when a web server is started. Files of any size can be securely shared. Because you host the files on your computer and use no third-party software, you don't have to worry about prying eyes.  
```

Comment:
Please add a linebreak in long paragraphs, so the translation framework generates strings easier to translate. 
This line for example could be instead:

> For sharing files over Tor, [OnionShare](https://onionshare.org/) is a good option.
> OnionShare is temporarily/stealthily made available as a Tor Onion Service when a web server is started.
> Files of any size can be securely shared.
> Because you host the files on your computer and use no third-party software, you don't have to worry about prying eyes.
Also, see the preferred link syntax. There are occurrences of html link but we are trying to remove them.